bin/validate-json

```diff
@@ -6,32 +6,18 @@
  * @author Christian Weiske <christian.weiske@netresearch.de>
  */
 
-/**
- * Dead simple autoloader
- *
- * @param string $className Name of class to load
- *
- * @return void
- */
-function __autoload($className)
-{
-    $className = ltrim($className, '\\');
-    $fileName  = '';
-    if ($lastNsPos = strrpos($className, '\\')) {
-        $namespace = substr($className, 0, $lastNsPos);
-        $className = substr($className, $lastNsPos + 1);
-        $fileName  = str_replace('\\', DIRECTORY_SEPARATOR, $namespace) . DIRECTORY_SEPARATOR;
-    }
-    $fileName .= str_replace('_', DIRECTORY_SEPARATOR, $className) . '.php';
-    if (stream_resolve_include_path($fileName)) {
-        require_once $fileName;
-    }
+// support running this tool from git checkout
+if (is_dir(__DIR__ . '/../vendor')) {
```

Comment:
Can you clarify? `PATH_SEPARATOR` has nothing to do with filesystem paths - it's the ':' character used as a list separator when defining multiple search paths (ie. what you'll find in the `$PATH` environment variable). Not sure how `dirname` has anything to do with that...